### ISSUE TITLE
Question about the result of sf-dfnet

### ISSUE THREAD

**Issue by Pedrojj97** [lxtGH/SFSegNets#12]
The miou of the dfv2 trained model  you released is 75.8%  on validation set while the miou is 77.8% on the test set reported in your paper,Would you mind to explain why the results are so different。

**Comment by lxtGH** [comment]
Good Question. SFnet on small model results better on test set(about 76.6 if I remembered correctly using dfv2 backbone) than val set on Cityscapes which shows its generation ability.  Moreover, affter appending val dataset, there is another about 0.8-1.0 mIoU gain on final results.

**Comment by Pedrojj97** [comment]
Thanks for your reply, so do you mean that the results of sfnet-dfv2 reported in the paper are trained  on both the train set and the val set? Then does the final result of sfnet-resnet18 reported in the paper trained on both the train and val set too.

**Comment by lxtGH** [comment]
For cityscapes, all the paper use train + val to report their results on test set.
